### ISSUE TITLE
Mouse Sensitivity is Super Slow

### ISSUE THREAD

**Issue by Conner1115** [Phi-001/WebGLGame#1]
I guess you are trying to make a first person shooter out of this.  Can you possibly increase the mouse sensitivity so that when I move my mouse, the world rotates faster?

**Comment by Phi-001** [comment]
Better now?

**Comment by Conner1115** [comment]
Yes, better, but still a bit slow.  Maybe add a range slider to be able to
control the sensitivity at which the camera rotates.  Very nice.
